src/lib/media-import/status.ts

```diff
@@ -350,12 +355,12 @@ Downloading errors details from ${ fileErrorsUrl }...
 		results: AppEnvironmentMediaImportStatus
 	) {
 		progressTracker.suffix += `${ chalk.yellow(
-			`⚠️  ${ fileErrors.length } file import error(s) were found`
+			`⚠️  ${ fileErrors.length } import error(s) were found`
 		) }`;
 
 		if ( ( results.filesTotal ?? 0 ) - ( results.filesProcessed ?? 0 ) !== fileErrors.length ) {
 			progressTracker.suffix += `. ${ chalk.italic.yellow(
-				'File import errors report size threshold reached.'
+				'Import errors report size threshold reached.'
```

Comment:
v-minor:

`Import errors report size threshold` is too big of a subject in the sentence. Perhaps we do something like
```suggestion
				'Import error-report is too large to ???'
```

Reply:
Import error-report is too large to ... ??

Could you elaborate please?

Reply:
actually I decided to revert my changes on this altogether because it is part of the v1 flow . And I didn't want to make a breaking change there.